### ISSUE TITLE
Marking Graph SDK+API Bugs as "Security Issues"

### ISSUE THREAD

**Issue by jebeld17** [microsoftgraph/msgraph-sdk-android#100]
With user privacy becoming more and more a priority every day and the sometimes-personally-identifiable information contained in GraphAPI error messages output when a service or app hits a snag (such as token codes, uuids, etc.), I believe it is necessary to have a "Mark as Security Issue" flag or checkbox to fill when a user is submitting a bug report for the MSGraph SDK and API in the official GitHub pages. This will prevent any PII (Personally Identifiable Information) from being publicly accessible. 

Tl;Dr:
Please bring a "Mark bug as 'security issue'" option to the MSGraph SDK and API GitHub Issue Tracker and Generator, to hide any PII from being publicly accessible.

**Comment by baywet** [comment]
Hi everyone,
Thank you for the contribution, we have moved away from an android specific SDK to a [Java SDK](https://github.com/microsoftgraph/msgraph-sdk-java) and this repo will be archived very soon.
If you're still facing the same issue with the Java SDK, feel free to re-open an issue over there.
Closing.
